rbac/management/migrations/0041_auditlogmodel.py

```diff
@@ -0,0 +1,65 @@
+# Generated by Django 4.1.7 on 2023-08-23 18:41
```

Comment:
it looks that there is one migration and some other migration which is changing the previous ones .  If you add one table, can you create just one final migration with adding one table ? remove/alter are needed only when database changes are in stage/production applied and we need to do some change.